android/src/main/java/com/tailscale/ipn/App.kt

```diff
@@ -412,26 +414,23 @@ open class UninitializedApp : Application() {
       return
     }
     val nm: NotificationManagerCompat = NotificationManagerCompat.from(this)
-    nm.notify(STATUS_NOTIFICATION_ID, buildStatusNotification(vpnRunning))
+    nm.notify(STATUS_NOTIFICATION_ID, buildStatusNotification())
   }
 
-  fun buildStatusNotification(vpnRunning: Boolean): Notification {
-    val message = getString(if (vpnRunning) R.string.connected else R.string.not_connected)
-    val icon = if (vpnRunning) R.drawable.ic_notification else R.drawable.ic_notification_disabled
```

Comment:
The drawables and strings for the disconnected state can possibly be removed from resources, since they're no longer used.